packages/yumemi_lints/pubspec.yaml

```diff
@@ -6,6 +6,15 @@ repository: https://github.com/yumemi-inc/flutter-yumemi-lints
 environment:
   sdk: '>=2.17.0 <4.0.0'
 
+dependencies:
+  args: ^2.3.1
+  meta: ^1.10.0
```

Comment:
![ask-badge](https://img.shields.io/badge/review-ask-yellowgreen.svg) 
It seems like using Meta 1.12.0 would be fine, but were there any issues?
https://pub.dev/packages/meta/versions

Reply:
@morikann 
Thank you for your kind response 🙏